Intermittent FAIL in `/webdriver/tests/classic/get_alert_text/get.py`

```
OK /webdriver/tests/classic/get_alert_text/get.py

    FAIL [expected PASS] subtest: test_get_confirm_text

    AssertionError: no such alert (404): No user prompt is currently active.

    FAIL [expected PASS] subtest: test_get_prompt_text

    AssertionError: assert 'Hello' == 'Enter Your Name: '
```

--- a/python/servo/try_parser.py
+++ b/python/servo/try_parser.py
@@ -124,12 +124,7 @@ def handle_preset(s: str) -> Optional[JobConfig]:
             "WebDriver",
             Workflow.LINUX,
             wpt=True,
-            wpt_args=" ".join(
-                [
-                    "./tests/wpt/tests/webdriver/tests/classic/",
-                    "--processes 1",
-                ]
-            ),
+            wpt_args="./tests/wpt/tests/webdriver/tests/classic/",
             unit_tests=False,
             number_of_wpt_chunks=1,
         )

--- a/python/servo/try_parser.py
+++ b/python/servo/try_parser.py
@@ -131,7 +131,7 @@ def handle_preset(s: str) -> Optional[JobConfig]:
                 ]
             ),
             unit_tests=False,
-            number_of_wpt_chunks=2,
+            number_of_wpt_chunks=1,
         )
     elif any(word in s for word in ["vello"]):
         return JobConfig(

